docs/adr/0032-l10n-i18n-improvements.md

```diff
@@ -0,0 +1,138 @@
+# L10n / I18N Improvements
+
+- Status: accepted
+- Deciders: Lauren Zugai, Dan Schomburg, Barry Chen, Vijay Budhram, Wil Clouser
+- Date: 2022-09-22
+
+## Technical Story
+
+[This document](https://docs.google.com/document/d/1TgD7WBLvS4aZ10neLJNk-EYvDR8Kb9GOk2KHxhjVzbk/) was a RFC from 2022-07-09 to 2022-09-19 and contains Jira links, more information, and more context.
+
+## Context and Problem Statement
+
+As Firefox Accounts has migrated from gettext to Fluent over the last 2-3 years, several questions around l10n, including our infrastructure (i18n) and consistency amongst our various packages, as well as discussions around general improvements have surfaced.
+
+While discussions occurred in the linked document above, this ADR captures major decisions from that doc, mentions other relevant past decisions, and only lists pros/cons, or negative consequences, of new decisions that have reasonable alternatives.
+
+Note that most decisions cover Fluent packages - `fxa-settings`, `fxa-payments-server`, `fxa-react`, and `fxa-auth-server`. As noted in the RFC, `fxa-content-server` still uses gettext and will share our Fluent infrastructure when we refactor it to React in 2023.
+
+## Decision Drivers
+
+- Consistency across packages
+- Reliability
+- More robust and complete testing
+- Allowing `fxa-react` to manage its own l10n
+- More straightforward l10n in emails
+
+## Decision outcome 1: Use consistent FTL paths, names, and setup across packages
+
+`en-US` was FxA's default locale until early 2021 when the l10n team recommended we switch to `en`. `fxa-settings` still contains `en-US.ftl` files that will be updated to `en.ftl`.
+
+Historically, we did not deploy our `en` or `en-US` Fluent bundles, but we have since May 2022 to support any missing fallback text and for consistency across locales and environments.
+
+### `fxa-payments-server` setup changes
```

Comment:
I'm confused by the hierarchy of the document.  You have a decision outcome as an `<h2>` above this subject, but the rest of the doc has the subjects above the decision outcomes and I think inconsistent h1/h2/h3 but that might just be the ordering.  It's probably worth revisiting that to help clarify the ideas here.

Reply:
Yeah, good point, I'll revisit this. The typical ADR format felt a little awkward for this since there were a lot of things to cover and it wasn't a 1:1 with the RFC. I knocked this down from 6-7 "decisions" made to 4, placing some stuff under others.

Reply:
I pushed an update here.